src/main/target/CC3D/target.h

```diff
@@ -72,8 +72,13 @@
 #define USE_VCP
 #define USE_UART1
 #define USE_UART3
+
+#ifdef CC3D_OPBL
```

Comment:
It is a problem with readability, in my opinion. It would force you to read the whole file to get an idea of the differences between CC3D and CC3D_OPBL.

My proposed format has config settings in 2 places at maximum (once how how it is for CC3D, and once how it is for CC3D_OPBL, if it is different from CC3D).

The double define that you fixed was within the same `#ifdef`, so adding more ifdefs would not fix that kind of oversight (which most likely happened as the result of two totally legitimate merges).